### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by pherum**
https://github.com/lipis/bootstrap-social/issues/116

### CODE REVIEW

File: bootstrap-social.less [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Social Buttons for Bootstrap
+ * Social Buttons for Bootstrap v5.0.0
```
Comment (lipis) [inline bootstrap-social.less:2]: this will need maintenance.. so I would suggest revert that one..

File: bootstrap-social.scss [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * Social Buttons for Bootstrap
+ * Social Buttons for Bootstrap v5.0.0
```
Comment (lipis) [inline bootstrap-social.scss:2]: same here

File: index.html [hunk 3]
```diff
@@ -283,7 +283,7 @@ <h3 class="text-center">Different Sizes</h3>
...
     <!-- Some JavaScript that is used only in this demo, not needed for the buttons -->
-    <script src="assets/js/jquery.js"></script>
+    <script src="assets/js/jquery.min.js"></script>
```
Comment (lipis) [inline index.html:286]: ?

Comment (pherum) [inline index.html:286]: @lipis не понял вопроса

**Comment by lipis** [comment]
#127

**Pull request closed without merging**